rpmlint/descriptions/PythonCheck.toml

```diff
@@ -0,0 +1,20 @@
+python-tests-in-package="""
+test/ or tests/ directory in package directory. Tests in %{python_sitelib}/<pkgname> should be packaged in a separate test or tests subpackage."""
+python-doc-in-package="""
+doc/ or docs/ directory in package directory. Documentation should go into %{docdir}, not %{python_sitelib}/<pkgname>"""
+python-egg-info-distutils-style="""
+The package's egg-info is a distutils style file, rather than a setuptools
+style directory. Please convert to setuptools style.
```

Comment:
The correct message we want to spread is to update to dist-info standardized core metadata rather than recommending the replacement of deprecated stuff with a bit less deprecated stuff.